dist/angular-storage.js

```diff
@@ -12,18 +12,25 @@ angular.module('angular-storage',
 
 angular.module('angular-storage.cookieStorage', [])
   .service('cookieStorage', ["$injector", function ($injector) {
-    var $cookieStore = $injector.get('$cookieStore');
+    var $cookies = $injector.get('$cookies');
```

Comment:
The most recent Angular documentation states:

"Note: The $cookieStore service is deprecated. Please use the $cookies service instead."